test/loader-common.js

```diff
@@ -870,7 +870,7 @@ export const LoaderCommonFactory = ({
       });
 
       QUnit.test('drops partIndex if playlist update drops parts', function(assert) {
-        assert.timeout(100000000000000000000);
+        loader.duration_ = () => Infinity;
```

Comment:
ie 11 has the duration as NaN here, so sync-controller gets a sync point of `1` for a vod video.